app/controllers/sessions_controller.rb

```diff
@@ -35,6 +35,7 @@ class SessionsController < ApplicationController
 
   # Login form
   def start
+    @banners = Banner.where('expires_at > ?', DateTime.now)
```

Comment:
Maybe make this into a scope to avoid repeating it.

Reply:
Done.